.changeset/curly-buttons-invent.md

```diff
@@ -0,0 +1,6 @@
+---
+"@fuel-ts/abi-coder": minor
+"@fuel-ts/account": patch
+---
+
+chore!: reorder `Interface.decodeLog` arguments
```

Comment:
Approving with a comment to hold auto-merge from kicking before conversations are settled.